src/scale-manager/README.md

```diff
@@ -1,23 +1,115 @@
 
 ```javascript
-var scaleManager = new ScaleManager(function(resizeData) {
-  console.log('This is called on window resize');
+let scaleManager = new ScaleManager({
+  width: 1320,
+  height: 780,
+  callback: function(resizeData) {
+    console.log('This is called on window resize');
 
-  console.log('Window width is', resizeData.width);
-  console.log('Window height is', resizeData.height);
-  console.log('Window aspect ratio is', resizeData.ratio);
+    console.log('game width is', resizeData.width);
+    console.log('game height is', resizeData.height);
+    console.log('game scale is', resizeData.scale);
+  }
 });
 ```
 
-The `enable` method exists in case you do not have a listener ready at the time of construction or need to replace the
-existing listener.
+The `enable` method exists in case you do not have a listener ready at the time of construction or need to replace the existing listener. 
 
 ```javascript
 scaleManager.enable(function(resizeData) {
   console.log('The old listener is replaced with this one!', resizeData);
 });
 ```
+
+**Note:** There is a 500ms delay added to the event to solve an issues where on some browsers the window isn't fully resized before the event fires.
+
+### Callback Example
+
+The scale manager callback parameter contains `width`, `height`, and `scale`. these values can be used to resize your game. Depending on the engine you choose you'll have to use them differently. `scale` contains `x` and `y`, these are the independent scaling values for the x and y axis.
+
+```javascript
+scaleManager.enable(({width, height, scale}) => {
+  // -- PIXI -- //
+  const view = this.pixi.view;
+  const stage = this.pixi.stage;
+  const renderer = this.pixi.renderer;
+
+  stage.position.set(renderer.width / 2, renderer.height / 2);
+  stage.scale.set(scale.x, scale.y);
+
+  stage.pivot.x = renderer.width / 2;
+  stage.pivot.y = renderer.height / 2;
+
+  view.style.width = width + 'px';
+  view.style.height = height + 'px';
+
+  view.style.position = 'absolute'
+  view.style.left = '0px';
+  view.style.top = '0px'; 
+});
+```
+
+**Note**: A Phaser3 example can be found here: https://github.com/SpringRoll/Springroll-Seed/tree/templates/pixi
+
+## Safe Scaling
+Safe scaling is enabled by setting `safeWidth` and `safeHeight` in the scaleManager's config. 
+
+```javascript
+let scaleManager = new ScaleManager({
+  width: 1320,
+  height: 780,
+  safeWidth: 1024,
+  safeHeight: 660,
+  callback: pixiResize
+});
+```
+
+This allows for the scale manager to calculate the canvas's target size and scale to ensure that the viewport's minimum is never smaller than `safeWidth` and `safeHeight`, and It's maximum never larger than `width` and `height`.
+
+## Scaled Entities
+
+Any object that needs to be updated after a resize event can be implemented as a scaled Entity. The object must have an `onResize` function.
+
+```javascript
+class MyEntity extends ScaledEntity
```

Comment:
I think there needs to be a stronger example here. Is this something we would bolt onto existing items, ala:

```javascript
PIXI.Container.prototype.onResize = function({ offset, scale, gameSize }) {
  this.position.x = offset.x;
  this.position.y = offset.y;
  this.scale.x = scale.x;
  this.scale.y = scale.y;
};
```

It's currently not clear to me what sort of framework-specific code that I as a developer would put in `onResize`.